staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -110,16 +73,9 @@ func loadWebhook(configFile string, groupVersion schema.GroupVersion) (*webhook.
 		[]schema.GroupVersion{groupVersion}, defaultInitialBackoff)
 }
 
-func newBlockingWebhook(configFile string, groupVersion schema.GroupVersion) (*blockingBackend, error) {
-	w, err := loadWebhook(configFile, groupVersion)
-	if err != nil {
-		return nil, err
-	}
-	return &blockingBackend{w}, nil
-}
-
 type blockingBackend struct {
```

Comment:
Just rename it to backend

Reply:
Since here, log/backend.go should also named to `backend` to keep consistent.